source/_patterns/00-atoms/forms/select.mustache

```diff
@@ -12,14 +12,16 @@
     {{#autofocus}} autofocus{{/autofocus}}
     {{#required}} required="required"{{/required}}
     {{#disabled}} disabled="disabled"{{/disabled}}
+    {{#userInputInvalid}} aria-invalid="true"{{/userInputInvalid}}
```

Comment:
Keeps the invalid state indication separate from the messaging logic, so an informational message can be provided when the form is not invalid.